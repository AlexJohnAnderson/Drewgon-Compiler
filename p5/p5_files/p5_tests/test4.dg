int a;
bool b;
int c;
void foo(){
a = 5;
b = true;
if(a < b){
c = a;

}
}

