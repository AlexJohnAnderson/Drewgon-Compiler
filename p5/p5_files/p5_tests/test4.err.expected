FATAL [7,8]-[7,9]: Relational operator applied to non-numeric operand
Type Analysis Failed
